### ISSUE TITLE
Decide which information to track in statistics events

### ISSUE THREAD

**Issue by max-moser** [inveniosoftware/invenio-app-rdm#2091]
PR https://github.com/inveniosoftware/invenio-app-rdm/pull/2080 integrates Invenio-Stats into InvenioRDM:
Every time a record's landing page is visited, a statistics event containing some information regarding the record and context is emitted, and this event is indexed (and aggregated) in the search engine.

Even though auto-mapping works in ES/OS, it can lead to fields being unusable for search purposes if the field's type is guessed incorrectly.
Thus, we need to think about which information we want to track in these events (e.g. the records' communities) and add the fields in the templates (and event builders).

**Comment by lnielsen** [comment]
My five cents:

- User ids/identity we cannot track due to privacy. 
- Communities: yes we want to track the UUIDs of the community
- Country would be a nice to have.
- We could potentially track referring domains

**Comment by fenekku** [comment]
@max-moser can we close this? I don't even see stats in invenio-app-rdm anymore and we seem to have a number of fields collected that work.

**Comment by max-moser** [comment]
Yeah, I think what we're currently tracking is probably good enough.
